icu4c/source/common/uresimp.h

```diff
@@ -270,11 +270,13 @@ ures_getByKeyWithFallback(const UResourceBundle *resB,
  * function can perform fallback on the sub-resources of the table.
  * @param resB              a resource
  * @param inKey             a key associated with the requested resource
+ * @param len               if not NULL, used to return the length of the string
```

Comment:
NULL/``null_ptr``, 

Also, I prefer  the following comment:
`@ param len            the length of the string`

and if there is no string, return 0

Reply:
I'd usually prefer "nullptr", but this is an old file and has lots of references to NULL - so I'll leave this as "NULL" for consistency.

Your suggestion is ambiguous: sounds like it's an input parameter rather than output.

I could say "returns the length of the string. Pass NULL if this is not needed" - but this feels unwieldy. I want to mention in the docs that len need not be provided, which is why I went with the "if not NULL" to start with. It sounds like my comment wasn't entirely clear to you though. I'm open to other suggestions as to how to make it clearer. :thinking: